doc-content/enterprise-only/project-data/preview-editor-data-objects-import-proc.adoc

```diff
@@ -0,0 +1,16 @@
+[id='preview-editor-data-objects-import-proc']
+= Importing data objects
+
+*Test Scenario (Preview)* editor will load all data objects that are located in the same package as the test scenario. You can view all the data objects from the *Data Objects* tab in the editor. The loaded data objects are also displayed in the *Test Editor* tab in the *Test Tools* panel.
+
+You need to close and reopen the preview editor in case the data objects change (for example, when a new data object is created or when an existing one is deleted). Select a data object from the list to display its fields and their types.
```

Comment:
"its" fields and "their" types? Shouldn't it be "its field and types"? Not sure what is "their" here.

Reply:
Sentence rephrased.